README.md

```diff
@@ -2,8 +2,8 @@
 
 Combine some great things under one umbrella.
 
- - Maven 3.5
- - Node 10 (via nvm)
- - Gulp 4 (via npm)
+ - Maven 3.6.3
+ - Java 15
+ - Node 14 (via nvm)
```

Comment:
This should be Node 15, right?

Reply:
Right. Fixed it.
I also brought back gulp which i removed on my version.